icu4c/source/test/fuzzer/break_iterator_fuzzer.cc

```diff
@@ -11,8 +11,8 @@ IcuEnvironment* env = new IcuEnvironment();
 
 extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size) {
   UErrorCode status = U_ZERO_ERROR;
-  icu::UnicodeString str(UnicodeStringFromUtf8(data, size));
-
+  icu::UnicodeString fuzzstr(reinterpret_cast<const UChar*>(data),
+                             static_cast<int>(size) / sizeof(UChar));
   auto rng = CreateRng(data, size);
   const icu::Locale& locale = GetRandomLocale(&rng);
 
```

Comment:
Instead of random, could use the first byte or two of the raw fuzz data to select the locale and break iterator type, and omit those byte(s) from the test string. Might let the fuzzer be more effective in directing the data permutations.

Reply:
Done.